### ISSUE TITLE
Cannot find module 'iconv'

### ISSUE THREAD

**Issue by uditalias** [dommmel/fill-pdf#6]
You forgot to add this dependency in your package.json

`npm install --save iconv`

**Comment by AdamBrodzinski** [comment]
+1 and also 'temp'

**Comment by dommmel** [comment]
Afaics both deps (iconv + temp) are there: https://github.com/dommmel/fill-pdf/blob/master/package.json#L26-L27

**Comment by AdamBrodzinski** [comment]
I'm not sure why they're not showing up in the package.json that gets downloaded (v0.0.2) but the file looks different than what's on master.

```
npm install fill-pdf
fill-pdf@0.0.2 node_modules/fill-pdf

pdfmaker|⇒ node app.js

module.js:340
    throw err;
          ^
Error: Cannot find module 'iconv'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/adam/projects/sandbox/pdfmaker/node_modules/fill-pdf/index.js:2:13)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

```
pdfmaker|⇒ cat node_modules/fill-pdf/package.json
{
  "name": "fill-pdf",
  "version": "0.0.2",
  "description": "Fill out pdf forms",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "repository": {
    "type": "git",
    "url": "git://github.com/dommmel/fill-pdf"
  },
  "keywords": [
    "pdf",
    "fdf",
    "utf8"
  ],
  "author": {
    "name": "Dominik Heier",
    "email": "dominik.heier@gmail.com",
    "url": "http://www.flexify.net/"
  },
  "license": "MIT",
  "bugs": {
    "url": "https://github.com/dommmel/fill-pdf/issues"
  },
  "homepage": "https://github.com/dommmel/fill-pdf",
  "_id": "fill-pdf@0.0.2",
  "dist": {
    "shasum": "21e1fb0484a14572a3fe008bce9cafc542e0a68b",
    "tarball": "http://registry.npmjs.org/fill-pdf/-/fill-pdf-0.0.2.tgz"
  },
  "_from": "fill-pdf@*",
  "_npmVersion": "1.3.24",
  "_npmUser": {
    "name": "dommmmel",
    "email": "dominik.heier@gmail.com"
  },
  "maintainers": [
    {
      "name": "dommmmel",
      "email": "dominik.heier@gmail.com"
    }
  ],
  "directories": {},
  "_shasum": "21e1fb0484a14572a3fe008bce9cafc542e0a68b",
  "_resolved": "https://registry.npmjs.org/fill-pdf/-/fill-pdf-0.0.2.tgz",
  "readme": "ERROR: No README data found!"
}
```

After adding in deps it works (pdf wasn't there so it errors but now runs)

```
pdfmaker|⇒ npm install --save iconv
-
> iconv@2.1.6 install /Users/adam/projects/sandbox/pdfmaker/node_modules/iconv
> node-gyp rebuild

  CC(target) Release/obj.target/libiconv/deps/libiconv/lib/iconv.o
  LIBTOOL-STATIC Release/iconv.a
  CXX(target) Release/obj.target/iconv/src/binding.o
  SOLINK_MODULE(target) Release/iconv.node
  SOLINK_MODULE(target) Release/iconv.node: Finished
iconv@2.1.6 node_modules/iconv
└── nan@1.6.2

pdfmaker|⇒ npm install --save temp

temp@0.8.1 node_modules/temp
└── rimraf@2.2.8

pdfmaker|⇒ node app.js
# this error is expected, pdf really isn't there
stderr: Error: Unable to find file.
```

**Comment by jspizziri** [comment]
+1

@dommmel the package deps are in the repo, but the npm package is out of date. Please tag your releases and republish to npm.

**Comment by jspizziri** [comment]
@AdamBrodzinski & @uditalias,

Until the package gets update you can install it directly from github via:

```
npm install git+https://github.com/dommmel/fill-pdf.git --save
```

**Comment by jspizziri** [comment]
@dommmel,

Also your packages are out of date, there was an issue with iconv https://github.com/bnoordhuis/node-iconv/issues/124 which is causing a build error.

**Comment by dommmel** [comment]
I just merged #8 and republished. Let me know if that resolves your issues.
